types/algoliasearch/index.d.ts

```diff
@@ -1104,7 +1105,7 @@ declare namespace algoliasearch {
      * 'min' Only keep results with the minimum number of typos
      * 'strict' Hits matching with 2 typos are not retrieved if there are some matching without typos.
      */
-    typoTolerance?: boolean;
+    typoTolerance?: boolean | string;
```

Comment:
thanks, this is correct! I've lowered the options to this enum though :)

```suggestion
    typoTolerance?: boolean | 'min' | 'strict';
```